src/Compilers/CSharp/Test/Semantic/Semantics/UnsafeTests.cs

```diff
@@ -3077,13 +3077,13 @@ public struct C
             CreateCompilation(text, options: TestOptions.UnsafeReleaseDll).VerifyDiagnostics(
                 // (13,20): error CS0208: Cannot take the address of, get the size of, or declare a pointer to a managed type ('A')
                 //             public A*[,][] aa; //CS0208
-                Diagnostic(ErrorCode.ERR_ManagedAddr, "A*").WithArguments("A"),
+                Diagnostic(ErrorCode.ERR_ManagedAddr, "aa").WithArguments("A"),
```

Comment:
>aa [](start = 55, length = 2)

Should the error be reported on the type rather than the field name?

Reply:
It would be nice if we could do this but it's tricky as we discussed offline.